loom/script/reflection/lsMemberInfo.h

```diff
@@ -384,6 +387,16 @@ class MemberInfo {
         this->type = type;
     }
 
+    inline bool getMissing()
+    {
+        return missing;
+    }
+
+    inline void setMissing(const char *message)
```

Comment:
Oh it's a subclass? Should this be virtual, ie, assert if call? Or just noted as such? Why inline?